tests/unittest_inference.py

```diff
@@ -541,18 +541,6 @@ class Warning(Warning):
         self.assertEqual(ancestor.root().name, BUILTINS)
         self.assertRaises(StopIteration, partial(next, ancestors))
 
```

Comment:
I agree these test is not specially clear, but i do not understand why do you delete it. Can you explain a little bit more?